recipes/libsecret/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Nasty hack for $PREFIX/bin/intltool-*
+ln -s "$PREFIX/bin/perl" "$PREFIX/bin/perl -w"
+
+./configure --prefix="$PREFIX" \
+    --disable-manpages \
+    --with-libgcrypt-prefix="$PREFIX"
+
+# Requires python-dbus and some other stuff
```

Comment:
How much "other stuff"? It'd be useful to have at least some basic check that things are working, either here or in the tests.

Reply:
Added a patch to disable some tests.